01_values.md

```diff
@@ -1,554 +1,277 @@
-{{meta {docid: values}}}
```

Comment:
oi, @diemoritat.

valeu demais pelo PR.

reparei que você removeu as linhas dos metadados. 

acredito que removendo essas informações, teremos problemas na hora de fazer o build do livro.

você poderia deixá-las jeito que estão?

_ps.  se for o caso, removemos/traduzimos esses dados no futuro, ok?_

Reply:
@joaostein tranquilo!
Acabei removendo essas infos porque achei que o livro ficaria no github como docs mesmo, mas já coloquei as tags de volta no arquivo :)